tampermonkey/PPT_AutoComplete.js

```diff
@@ -296,6 +296,13 @@ const primaryContactEmailAddress = () => {
     }
 }
 
+const primaryContactEmailAddressPasscode = () => {
+    if (currentPageIs('/plastic-packaging-tax/primary-contact-email-passcode')) {
+
+        document.getElementById('value').value = 'DNCLRK'
```

Comment:
How does this work if the test endpoint isn't ready yet? Is this just placeholder script or does this work as things stand?

Reply:
This is not ready just a place holder